Solutions/Dev 0270 Detection and Hunting/Analytic Rules/Dev-0270WMICDiscoverySep2022.yaml

```diff
@@ -24,7 +24,7 @@ query: |
   (union isfuzzy=true
   (SecurityEvent
   | where EventID==4688
-  | where CommandLine has "wmic computersystem get domain" and ParentProcessName =~ "dllhost.exe"
+  | where CommandLine has "wmic computersystem get domain" and tostring(split(ParentProcessName, @'\')[-1]) =~ "dllhost.exe"
   | project TimeGenerated, HostCustomEntity = Computer, AccountCustomEntity = Account, AccountDomain, ProcessName, ProcessNameFullPath = NewProcessName, EventID, Activity, CommandLine, EventSourceName, Type
   ),
   (DeviceProcessEvents 
```

Comment:
What kind of values coming under 'InitiatingProcessFileName' and 'InitiatingProcessCommandLine'. Don't we need similar change here too?

Reply:
I have to say I don't know how this rule was developed, I do not understand why InitiatingProcessFileName AND InitiatingProcessCommandLine need to be the value ```dllhost.exe```.

In my own logs InitiatingProcessCommandLine always has the value

```DllHost.exe /Processid:{00000000-0000-0000-0000-000000000000}```

but that does not mean that it happens also in the desired events to be detected.

Reply:
Still I have made the change as you suggested.